src/rgw/rgw_rados.cc

```diff
@@ -10645,13 +10645,13 @@ int RGWRados::Object::Read::iterate(int64_t ofs, int64_t end, RGWGetDataCB *cb)
   while (!done) {
     r = data->wait_next_io(&done);
     if (r < 0) {
-      dout(10) << "get_obj_iterate() r=" << r << ", canceling all io" << dendl;
+      dout(10) << __func__ << " r=" << r << ", cancelling all io" << dendl;
```

Comment:
nit: `canceling` is also a correct spelling, so i'd prefer not to change it

Reply:
@cbodley `canceling` is American and `cancelling` is British. So for people on the Eastern side of the world `cancelling` looks correct :-D
https://english.stackexchange.com/questions/42745/canceling-or-cancelling

I think, let's follow American, as cbodley suggests.